### PR TITLE
[G3] Driver-only cipher+aead: TLS: compat.sh 

### PR DESCRIPTION
**Pull request by lpy4105**
## Description

Resolve: #6324 

## PR checklist

- [x] **changelog** not required
- [x] **backport** not required, new feature.
- [x] **tests** provided

### CODE REVIEW

**Comment by lpy4105** [comment]
OpenCI passed, this PR is ready for review.

**Review by mpg — DISMISSED** [review]
LGTM

**Review by valeriosetti — DISMISSED** [review]
;)

File: tests/scripts/all.sh [hunk 1]
```diff
@@ -3774,6 +3777,9 @@ component_test_psa_crypto_config_reference_cipher_aead () {
 
     msg "ssl-opt: crypto config with non-accelerated cipher and AEAD"
     tests/ssl-opt.sh
+
+    msg "compat.sh: crypto config with non-accelerated cipher and AEAD"
```
Comment (mpg) [inline tests/scripts/all.sh:3781]: Ah, actually, pre-existing issue: this should say "full config", as well as all other messages that currently have "crypto config" in those two components. I guess we forgot to change that when we expanded the config (which was initially crypto only indeed).

Could you fix it in this PR?

Comment (mpg) [reply tests/scripts/all.sh:3781]: Changed my mind let's merge the PR as is since it's approved.

Comment (gilles-peskine-arm) [reply tests/scripts/all.sh:3781]: Merges are blocked right now due to an incident on readthedocs (probably a configuration change we need to make), so you might as well update.

**Review by valeriosetti — APPROVED** [review]

**Review by mpg — APPROVED** [review]
LGTM

**Pull request merged**